locales/en/cos-ui.json

```diff
@@ -223,5 +223,7 @@
   "version": "Version",
   "viewJSONFormat": "View JSON format",
   "viewLess": "view less",
-  "viewMore": "view more"
-}
\ No newline at end of file
+  "viewMore": "view more",
+  "clusterid": "Cluster Id",
```

Comment:
This translation already exists [here](https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/main/locales/en/cos-ui.json#L36)